app/build.gradle

```diff
@@ -31,6 +34,7 @@ dependencies {
     exclude group: 'com.android.support', module: 'support-annotations'
   })
   compile 'com.android.support:appcompat-v7:26.+'
-  compile 'com.android.support.constraint:constraint-layout:1.0.2'
+  compile 'com.android.support:design:26.+'
+  compile 'com.android.support:support-vector-drawable:26.+'
```

Comment:
WARNING: [GradleDynamicVersion] Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (com.android.support:support-vector-drawable:26.+)
 Using `+` in dependencies lets you automatically pick up the latest available version rather than a specific, named version. However, this is not recommended; your builds are not repeatable; you may have tested with a slightly different version than what the build server used. (Using a dynamic version as the major version number is more problematic than using it in the minor version position.)